modules/nf-core/parabricks/haplotypecaller/tests/nextflow.config

```diff
@@ -0,0 +1,5 @@
+process {
+    withName: 'PARABRICKS_HAPLOTYPECALLER' {
+        ext.args = "--gvcf"
```

Comment:
maybe we can add a low-memory option here

Reply:
I could add --htvc-low-memory flag too

Reply:
Done